model/NotifierObserver.inc.php

```diff
@@ -152,17 +143,22 @@ public static function notify($event, $type, $ids, $extraData) {
 						$message[$key] = $val;
 					}
 				}
-				self::send($message);
+				foreach (self::$messageReceivers as $receiver) {
+					$receiver(
+						$apiKey,
```

Comment:
Let's take this opportunity to start hashing these on both sides — there's no reason to send raw API keys through the queue.